geemusic/utils/music_queue.py

```diff
@@ -59,5 +62,15 @@ def reset(self, tracks=[]):
         else:
             return self.song_ids[self.current_index]
 
+    def shuffleMode(self, value, ogsong_ids=[]):
```

Comment:
could you rename this to `enable_shuffle(self, value)`. I'd prefer to be consistent with `snake_case` rather than `camelCase` for methods.

it doesn't look like `og_song_ids` is being used here, so let's remove that as well.